docs/about_sat/dependencies.md

```diff
@@ -40,7 +40,7 @@ subcommand. Each service or component is listed under the product it belongs to.
 - Kubernetes
 - S3
 
-### COS
+### HPE Cray Supercomputing Compute Node Software Environment
```

Comment:
Looks like this dependency will be part of CNE (the extension pack) going forward: https://rndwiki-pro.its.hpecorp.net/display/HPCSCD/Initiative%3A+COS+software+feature+evaluation#Initiative:COSsoftwarefeatureevaluation-nmd